<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 References &amp; External Resources | Protein–Protein Interaction (PPI) Prediction Methods Summary</title>
  <meta name="description" content="A summary of computational methods for protein–protein interaction (PPI) prediction, including docking-based approaches, structural analysis, sequence/knowledge-based models, deep learning frameworks, and structure-generation models." />
  <meta name="generator" content="bookdown 0.45 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 References &amp; External Resources | Protein–Protein Interaction (PPI) Prediction Methods Summary" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A summary of computational methods for protein–protein interaction (PPI) prediction, including docking-based approaches, structural analysis, sequence/knowledge-based models, deep learning frameworks, and structure-generation models." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 References &amp; External Resources | Protein–Protein Interaction (PPI) Prediction Methods Summary" />
  
  <meta name="twitter:description" content="A summary of computational methods for protein–protein interaction (PPI) prediction, including docking-based approaches, structural analysis, sequence/knowledge-based models, deep learning frameworks, and structure-generation models." />
  

<meta name="author" content="Haoyi Yang" />


<meta name="date" content="2025-11-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="structure-generation.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PPI Prediction Methods</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction to Protein–Protein Interaction (PPI) Prediction</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#why-computational-ppi-prediction-matters"><i class="fa fa-check"></i>1.1 Why Computational PPI Prediction Matters</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#categories-of-computational-ppi-prediction-methods"><i class="fa fa-check"></i>1.2 Categories of Computational PPI Prediction Methods</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#docking-based-structure-based-methods"><i class="fa fa-check"></i><strong>1. Docking-Based (Structure-Based) Methods</strong></a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#structure-based-interface-analysis"><i class="fa fa-check"></i><strong>2. Structure-Based Interface Analysis</strong></a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#sequence-knowledgebased-prediction"><i class="fa fa-check"></i><strong>3. Sequence / Knowledge–Based Prediction</strong></a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#deep-learningbased-ppi-prediction"><i class="fa fa-check"></i><strong>4. Deep Learning–Based PPI Prediction</strong></a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#structure-generation-approaches"><i class="fa fa-check"></i><strong>5. Structure Generation Approaches</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#when-to-use-each-category"><i class="fa fa-check"></i>1.3 When to Use Each Category</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#structure-of-this-book"><i class="fa fa-check"></i>1.4 Structure of This Book</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#suggested-reading"><i class="fa fa-check"></i>1.5 Suggested Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="docking.html"><a href="docking.html"><i class="fa fa-check"></i><b>2</b> Docking-Based (Structure-Based) PPI Prediction</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#overview-of-docking-approaches"><i class="fa fa-check"></i>2.1 Overview of Docking Approaches</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#haddock"><i class="fa fa-check"></i>2.2 HADDOCK</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#hdock-server"><i class="fa fa-check"></i>2.3 HDOCK Server</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features-1"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input-1"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output-1"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages-1"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations-1"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#cluspro"><i class="fa fa-check"></i>2.4 ClusPro</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features-2"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input-2"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output-2"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages-2"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations-2"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#zdock"><i class="fa fa-check"></i>2.5 ZDOCK</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features-3"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input-3"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output-3"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages-3"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations-3"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#gramm-vakser-lab"><i class="fa fa-check"></i>2.6 GRAMM (Vakser Lab)</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features-4"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input-4"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output-4"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages-4"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations-4"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#pydock"><i class="fa fa-check"></i>2.7 pyDock</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#key-features-5"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#input-5"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#output-5"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#advantages-5"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#limitations-5"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#comparative-summary-of-docking-tools"><i class="fa fa-check"></i>2.8 Comparative Summary of Docking Tools</a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#practical-workflow-suggestions"><i class="fa fa-check"></i>2.9 Practical Workflow Suggestions</a>
<ul>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#when-starting-without-structural-information"><i class="fa fa-check"></i><strong>When starting without structural information</strong></a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#when-interface-residues-are-known-or-predicted"><i class="fa fa-check"></i><strong>When interface residues are known or predicted</strong></a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#when-many-protein-pairs-must-be-screened"><i class="fa fa-check"></i><strong>When many protein pairs must be screened</strong></a></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#when-sequences-are-available-but-structures-are-missing"><i class="fa fa-check"></i><strong>When sequences are available but structures are missing</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="docking.html"><a href="docking.html#relation-to-other-chapters"><i class="fa fa-check"></i>2.10 Relation to Other Chapters</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="structure-analysis.html"><a href="structure-analysis.html"><i class="fa fa-check"></i><b>3</b> Structure-Based Interface Analysis (PDBePISA)</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#purpose-of-structure-based-interface-analysis"><i class="fa fa-check"></i>3.1 Purpose of Structure-Based Interface Analysis</a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#what-is-pdbepisa"><i class="fa fa-check"></i>3.2 What Is PDBePISA?</a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#how-pdbepisa-works"><i class="fa fa-check"></i>3.3 How PDBePISA Works</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#buried-surface-area-bsa"><i class="fa fa-check"></i><strong>1. Buried Surface Area (BSA)</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#solvation-free-energy-gain-%CE%B4gint"><i class="fa fa-check"></i><strong>2. Solvation Free Energy Gain (ΔG<sub>int</sub>)</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#hydrogen-bond-and-salt-bridge-networks"><i class="fa fa-check"></i><strong>3. Hydrogen Bond and Salt Bridge Networks</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#interface-residue-mapping"><i class="fa fa-check"></i><strong>4. Interface Residue Mapping</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#biological-assembly-prediction"><i class="fa fa-check"></i><strong>5. Biological Assembly Prediction</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#key-outputs-of-pdbepisa"><i class="fa fa-check"></i>3.4 Key Outputs of PDBePISA</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#interface-summary"><i class="fa fa-check"></i><strong>• Interface Summary</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#assembly-summary"><i class="fa fa-check"></i><strong>• Assembly Summary</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#stability-prediction"><i class="fa fa-check"></i><strong>• Stability Prediction</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#detailed-contact-map"><i class="fa fa-check"></i><strong>• Detailed Contact Map</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#when-to-use-pdbepisa"><i class="fa fa-check"></i>3.5 When to Use PDBePISA</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#after-docking-chapter-2"><i class="fa fa-check"></i><strong>1. After Docking (Chapter 2)</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#after-alphafold-multimer-prediction-chapter-6"><i class="fa fa-check"></i><strong>2. After AlphaFold-Multimer Prediction (Chapter 6)</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#crystal-structure-interpretation"><i class="fa fa-check"></i><strong>3. Crystal Structure Interpretation</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#identifying-potential-hotspots"><i class="fa fa-check"></i><strong>4. Identifying Potential Hotspots</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#pdbepisa-vs.-docking-scores"><i class="fa fa-check"></i>3.6 PDBePISA vs. Docking Scores</a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#practical-workflow-integration"><i class="fa fa-check"></i>3.7 Practical Workflow Integration</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#a-recommended-pipeline-integrating-chapter-2-and-3"><i class="fa fa-check"></i><strong>A recommended pipeline integrating Chapter 2 and 3:</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#advantages-and-limitations"><i class="fa fa-check"></i>3.8 Advantages and Limitations</a>
<ul>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#advantages-6"><i class="fa fa-check"></i><strong>Advantages</strong></a></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#limitations-6"><i class="fa fa-check"></i><strong>Limitations</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-analysis.html"><a href="structure-analysis.html#summary"><i class="fa fa-check"></i>3.9 Summary</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html"><i class="fa fa-check"></i><b>4</b> Sequence- and Knowledge-Based PPI Prediction Models</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#overview-of-sequence-knowledge-based-methods"><i class="fa fa-check"></i>4.1 Overview of Sequence / Knowledge-Based Methods</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#evolutionary-features"><i class="fa fa-check"></i><strong>4.1.1. Evolutionary Features</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#functional-annotation"><i class="fa fa-check"></i><strong>4.1.2. Functional Annotation</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#literature-mining-text-evidence"><i class="fa fa-check"></i><strong>4.1.3. Literature Mining &amp; Text Evidence</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#biological-network-topology"><i class="fa fa-check"></i><strong>4.1.4. Biological Network Topology</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#homology-based-templates"><i class="fa fa-check"></i><strong>4.1.5. Homology-Based Templates</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#string"><i class="fa fa-check"></i>4.2 STRING</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#key-features-6"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#input-6"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#output-6"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#advantages-7"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#limitations-7"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#preppi"><i class="fa fa-check"></i>4.3 PrePPI</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#key-features-7"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#input-7"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#output-7"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#advantages-8"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#limitations-8"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#ppi-miner"><i class="fa fa-check"></i>4.4 PPI-Miner</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#key-features-8"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#input-8"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#output-8"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#advantages-9"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#limitations-9"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#coth"><i class="fa fa-check"></i>4.5 COTH</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#key-features-9"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#input-9"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#output-9"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#advantages-10"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#limitations-10"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#comparison-of-sequence-knowledge-based-tools"><i class="fa fa-check"></i>4.6 Comparison of Sequence / Knowledge-Based Tools</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#when-to-use-sequence-knowledge-based-methods"><i class="fa fa-check"></i>4.7 When to Use Sequence / Knowledge-Based Methods</a>
<ul>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#proteome-wide-filtering"><i class="fa fa-check"></i><strong>4.7.1. Proteome-wide filtering</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#finding-candidates-for-experimental-validation"><i class="fa fa-check"></i><strong>4.7.2. Finding candidates for experimental validation</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#supporting-evidence"><i class="fa fa-check"></i><strong>4.7.3. Supporting evidence</strong></a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#when-structural-data-is-unavailable"><i class="fa fa-check"></i><strong>4.7.4. When structural data is unavailable</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#integrating-knowledge-based-tools-into-a-workflow"><i class="fa fa-check"></i>4.8 Integrating Knowledge-Based Tools into a Workflow</a></li>
<li class="chapter" data-level="" data-path="sequence-knowledge.html"><a href="sequence-knowledge.html#summary-1"><i class="fa fa-check"></i>4.9 Summary</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="deep-learning.html"><a href="deep-learning.html"><i class="fa fa-check"></i><b>5</b> Deep Learning–Based PPI Prediction</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#why-deep-learning-for-ppi-prediction"><i class="fa fa-check"></i>5.1 Why Deep Learning for PPI Prediction?</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#advantages-of-dl-based-ppi-prediction"><i class="fa fa-check"></i>Advantages of DL-based PPI prediction</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#limitations-11"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#general-deep-learning-paradigms-for-ppi-prediction"><i class="fa fa-check"></i>5.2 General Deep Learning Paradigms for PPI Prediction</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#siamese-twin-network-models"><i class="fa fa-check"></i><strong>Siamese / Twin Network Models</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#graph-neural-network-gnn-models"><i class="fa fa-check"></i><strong>Graph Neural Network (GNN) Models</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#transformer-based-embedding-models"><i class="fa fa-check"></i><strong>Transformer-Based Embedding Models</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#contact-map-prediction-models"><i class="fa fa-check"></i><strong>Contact Map Prediction Models</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#d-script"><i class="fa fa-check"></i>5.3 D-SCRIPT</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#repository"><i class="fa fa-check"></i>Repository</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#key-features-10"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#model-architecture"><i class="fa fa-check"></i>Model Architecture</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#input-10"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#output-10"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#advantages-11"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#limitations-12"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#deeppurpose-ppi-module"><i class="fa fa-check"></i>5.4 DeepPurpose (PPI Module)</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#repository-1"><i class="fa fa-check"></i>Repository</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#key-features-11"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#input-11"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#output-11"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#advantages-12"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#limitations-13"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#mape-ppi"><i class="fa fa-check"></i>5.5 MAPE-PPI</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#repository-2"><i class="fa fa-check"></i>Repository</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#key-features-12"><i class="fa fa-check"></i>Key Features</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#model-architecture-1"><i class="fa fa-check"></i>Model Architecture</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#input-12"><i class="fa fa-check"></i>Input</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#output-12"><i class="fa fa-check"></i>Output</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#advantages-13"><i class="fa fa-check"></i>Advantages</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#limitations-14"><i class="fa fa-check"></i>Limitations</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#comparison-of-deep-learning-models"><i class="fa fa-check"></i>5.6 Comparison of Deep Learning Models</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#when-to-use-deep-learning-models"><i class="fa fa-check"></i>5.7 When to Use Deep Learning Models</a>
<ul>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#proteome-wide-screening"><i class="fa fa-check"></i><strong>5.7.1. Proteome-wide screening</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#dataset-driven-tasks"><i class="fa fa-check"></i><strong>5.7.2. Dataset-driven tasks</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#supporting-evidence-1"><i class="fa fa-check"></i><strong>5.7.3. Supporting evidence</strong></a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#cases-lacking-structural-data"><i class="fa fa-check"></i><strong>5.7.4. Cases lacking structural data</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#integrating-deep-learning-with-other-prediction-methods"><i class="fa fa-check"></i>5.8 Integrating Deep Learning with Other Prediction Methods</a></li>
<li class="chapter" data-level="" data-path="deep-learning.html"><a href="deep-learning.html#summary-2"><i class="fa fa-check"></i>5.9 Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="structure-generation.html"><a href="structure-generation.html"><i class="fa fa-check"></i><b>6</b> Structure Generation Methods: AlphaFold-Multimer</a>
<ul>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#what-is-alphafold-multimer"><i class="fa fa-check"></i>6.1 What Is AlphaFold-Multimer?</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#how-alphafold-multimer-works"><i class="fa fa-check"></i>6.2 How AlphaFold-Multimer Works</a>
<ul>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#joint-msa-construction"><i class="fa fa-check"></i><strong>6.2.1. Joint MSA Construction</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#evoformer-multimer-blocks"><i class="fa fa-check"></i><strong>6.2.2. Evoformer-Multimer Blocks</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#structure-module"><i class="fa fa-check"></i><strong>6.2.3. Structure Module</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#recycling"><i class="fa fa-check"></i><strong>6.2.4. Recycling</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#key-confidence-metrics-plddt-ptm-iptm"><i class="fa fa-check"></i>6.3 Key Confidence Metrics (pLDDT, pTM, ipTM)</a>
<ul>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#plddt-local-confidence"><i class="fa fa-check"></i><strong>pLDDT — Local Confidence</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#ptm-predicted-tm-score"><i class="fa fa-check"></i><strong>pTM — Predicted TM-score</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#iptm-inter-chain-tm-score"><i class="fa fa-check"></i><strong>ipTM — Inter-chain TM-score</strong></a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#recommended-composite-score"><i class="fa fa-check"></i><strong>Recommended composite score:</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#predicted-aligned-error-pae"><i class="fa fa-check"></i>6.4 Predicted Aligned Error (PAE)</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#evaluating-af-multimer-predictions-with-dockq"><i class="fa fa-check"></i>6.5 Evaluating AF-Multimer Predictions with DockQ</a>
<ul>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#dockq-score-interpretation"><i class="fa fa-check"></i><strong>DockQ score interpretation:</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#advantages-of-alphafold-multimer"><i class="fa fa-check"></i>6.6 Advantages of AlphaFold-Multimer</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#limitations-of-alphafold-multimer"><i class="fa fa-check"></i>6.7 Limitations of AlphaFold-Multimer</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#integrating-af-multimer-with-docking-pisa-and-deep-learning"><i class="fa fa-check"></i>6.8 Integrating AF-Multimer with Docking, PISA, and Deep Learning</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#recommended-best-practices"><i class="fa fa-check"></i>6.9 Recommended Best Practices</a></li>
<li class="chapter" data-level="" data-path="structure-generation.html"><a href="structure-generation.html#summary-3"><i class="fa fa-check"></i>6.10 Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>7</b> References &amp; External Resources</a>
<ul>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#major-ppi-databases"><i class="fa fa-check"></i>7.1 Major PPI Databases</a>
<ul>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#general-ppi-databases"><i class="fa fa-check"></i><strong>General PPI Databases</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#specialized-ppi-and-interaction-databases"><i class="fa fa-check"></i><strong>Specialized PPI and Interaction Databases</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#structural-databases-for-ppi-analysis"><i class="fa fa-check"></i>7.2 Structural Databases for PPI Analysis</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#benchmark-datasets-for-ppi-prediction"><i class="fa fa-check"></i>7.3 Benchmark Datasets for PPI Prediction</a>
<ul>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#docking-benchmarks"><i class="fa fa-check"></i><strong>Docking Benchmarks</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#deep-learning-benchmarks"><i class="fa fa-check"></i><strong>Deep Learning Benchmarks</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#sequence-based-prediction-benchmarks"><i class="fa fa-check"></i><strong>Sequence-Based Prediction Benchmarks</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#tools-and-servers-summary"><i class="fa fa-check"></i>7.4 Tools and Servers Summary</a>
<ul>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#docking-tools-chapter-2"><i class="fa fa-check"></i><strong>Docking Tools (Chapter 2)</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#structure-analysis-chapter-3"><i class="fa fa-check"></i><strong>Structure Analysis (Chapter 3)</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#sequence-knowledgebased-models-chapter-4"><i class="fa fa-check"></i><strong>Sequence / Knowledge–Based Models (Chapter 4)</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#deep-learning-models-chapter-5"><i class="fa fa-check"></i><strong>Deep Learning Models (Chapter 5)</strong></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#structure-generation-tools-chapter-6"><i class="fa fa-check"></i><strong>Structure-Generation Tools (Chapter 6)</strong></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html#references-1"><i class="fa fa-check"></i>7.5 References</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/HayesYoung58/Protein-Protein-Interaction-PPI-Prediction-Methods-Summary" target="_blank">View source on GitHub</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Protein–Protein Interaction (PPI) Prediction Methods Summary</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 hasAnchor" number="7">
<h1><span class="header-section-number">Chapter 7</span> References &amp; External Resources<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>This chapter provides a consolidated list of resources useful for protein–protein interaction (PPI) research, including databases, benchmark datasets, computational tools, and recommended literature.<br />
These resources support the methodologies described in Chapters 1–6.</p>
<hr />
<div id="major-ppi-databases" class="section level2 unnumbered hasAnchor">
<h2>7.1 Major PPI Databases<a href="references.html#major-ppi-databases" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>These databases curate experimentally validated or computationally predicted PPIs.</p>
<div id="general-ppi-databases" class="section level3 unnumbered hasAnchor">
<h3><strong>General PPI Databases</strong><a href="references.html#general-ppi-databases" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="33%" />
<col width="41%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Database</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>STRING</strong></td>
<td>Integrated PPI database combining experiments, text-mining, co-expression, and genomics</td>
<td><a href="https://string-db.org" class="uri">https://string-db.org</a></td>
</tr>
<tr class="even">
<td><strong>BioGRID</strong></td>
<td>Curated experimental PPI database</td>
<td><a href="https://thebiogrid.org" class="uri">https://thebiogrid.org</a></td>
</tr>
<tr class="odd">
<td><strong>IntAct</strong></td>
<td>EBI repository for experimentally validated PPIs</td>
<td><a href="https://www.ebi.ac.uk/intact" class="uri">https://www.ebi.ac.uk/intact</a></td>
</tr>
<tr class="even">
<td><strong>DIP</strong></td>
<td>Database of Interaction Proteins</td>
<td><a href="http://dip.doe-mbi.ucla.edu" class="uri">http://dip.doe-mbi.ucla.edu</a></td>
</tr>
<tr class="odd">
<td><strong>HINT</strong></td>
<td>High-quality interactomes across species</td>
<td><a href="http://hint.yulab.org" class="uri">http://hint.yulab.org</a></td>
</tr>
<tr class="even">
<td><strong>MINT</strong></td>
<td>Molecular Interaction database</td>
<td><a href="https://mint.bio.uniroma1.it" class="uri">https://mint.bio.uniroma1.it</a></td>
</tr>
<tr class="odd">
<td><strong>HuRI</strong></td>
<td>Human Reference Interactome (Y2H-based)</td>
<td><a href="https://interactome.baderlab.org" class="uri">https://interactome.baderlab.org</a></td>
</tr>
</tbody>
</table>
</div>
<div id="specialized-ppi-and-interaction-databases" class="section level3 unnumbered hasAnchor">
<h3><strong>Specialized PPI and Interaction Databases</strong><a href="references.html#specialized-ppi-and-interaction-databases" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="41%" />
<col width="33%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Database</th>
<th>Focus</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>STRING Viruses</strong></td>
<td>Viral-host PPIs</td>
<td><a href="https://viruses.string-db.org" class="uri">https://viruses.string-db.org</a></td>
</tr>
<tr class="even">
<td><strong>CORUM</strong></td>
<td>Protein complexes in human/mouse</td>
<td><a href="https://mips.helmholtz-muenchen.de/corum" class="uri">https://mips.helmholtz-muenchen.de/corum</a></td>
</tr>
<tr class="odd">
<td><strong>3did</strong></td>
<td>Domain–domain interactions</td>
<td><a href="https://3did.irbbarcelona.org" class="uri">https://3did.irbbarcelona.org</a></td>
</tr>
<tr class="even">
<td><strong>iRefWeb</strong></td>
<td>Integrated interaction resource</td>
<td><a href="http://irefindex.org" class="uri">http://irefindex.org</a></td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
<div id="structural-databases-for-ppi-analysis" class="section level2 unnumbered hasAnchor">
<h2>7.2 Structural Databases for PPI Analysis<a href="references.html#structural-databases-for-ppi-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="33%" />
<col width="41%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Database</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>PDB</strong></td>
<td>Worldwide Protein Data Bank</td>
<td><a href="https://www.rcsb.org" class="uri">https://www.rcsb.org</a></td>
</tr>
<tr class="even">
<td><strong>PDBePISA</strong></td>
<td>Interface &amp; assembly analysis</td>
<td><a href="https://www.ebi.ac.uk/pdbe/pisa" class="uri">https://www.ebi.ac.uk/pdbe/pisa</a></td>
</tr>
<tr class="odd">
<td><strong>HSSP</strong></td>
<td>Homology-derived structures</td>
<td><a href="https://swift.cmbi.umcn.nl/gv/hssp" class="uri">https://swift.cmbi.umcn.nl/gv/hssp</a></td>
</tr>
<tr class="even">
<td><strong>IBIS</strong></td>
<td>PPI structural templates</td>
<td><a href="https://www.ncbi.nlm.nih.gov/Structure/ibis/ibis.cgi" class="uri">https://www.ncbi.nlm.nih.gov/Structure/ibis/ibis.cgi</a></td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="benchmark-datasets-for-ppi-prediction" class="section level2 unnumbered hasAnchor">
<h2>7.3 Benchmark Datasets for PPI Prediction<a href="references.html#benchmark-datasets-for-ppi-prediction" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Benchmark datasets are essential for evaluating PPI models, docking pipelines, and multimer prediction.</p>
<div id="docking-benchmarks" class="section level3 unnumbered hasAnchor">
<h3><strong>Docking Benchmarks</strong><a href="references.html#docking-benchmarks" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="31%" />
<col width="43%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Dataset</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Docking Benchmark v5.0 (DB5)</strong></td>
<td>Gold-standard dataset for protein docking</td>
<td><a href="https://zlab.umassmed.edu/benchmark" class="uri">https://zlab.umassmed.edu/benchmark</a></td>
</tr>
<tr class="even">
<td><strong>CAPRI</strong></td>
<td>Community-wide docking assessment</td>
<td><a href="https://www.ebi.ac.uk/pdbe/complex-pred/capri" class="uri">https://www.ebi.ac.uk/pdbe/complex-pred/capri</a></td>
</tr>
</tbody>
</table>
</div>
<div id="deep-learning-benchmarks" class="section level3 unnumbered hasAnchor">
<h3><strong>Deep Learning Benchmarks</strong><a href="references.html#deep-learning-benchmarks" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="31%" />
<col width="43%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Dataset</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>DIPS</strong></td>
<td>42,000 PPI complexes for ML training</td>
<td><a href="https://github.com/phillipburgess/DIPS" class="uri">https://github.com/phillipburgess/DIPS</a></td>
</tr>
<tr class="even">
<td><strong>SKEMPI</strong></td>
<td>ΔΔG mutation effects on PPIs</td>
<td><a href="https://life.bsc.es/pid/skempi" class="uri">https://life.bsc.es/pid/skempi</a></td>
</tr>
<tr class="odd">
<td><strong>STRING-ML / BioGRID-ML</strong></td>
<td>ML-ready PPI datasets from STRING/BioGRID</td>
<td>(various GitHub repos)</td>
</tr>
</tbody>
</table>
</div>
<div id="sequence-based-prediction-benchmarks" class="section level3 unnumbered hasAnchor">
<h3><strong>Sequence-Based Prediction Benchmarks</strong><a href="references.html#sequence-based-prediction-benchmarks" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="38%" />
<col width="34%" />
<col width="26%" />
</colgroup>
<thead>
<tr class="header">
<th>Dataset</th>
<th>Focus</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>HopPPI</strong></td>
<td>Sequence-based PPI benchmarks</td>
<td><a href="https://github.com/caoyx1999/HopPPI" class="uri">https://github.com/caoyx1999/HopPPI</a></td>
</tr>
<tr class="even">
<td><strong>HHRI Dataset</strong></td>
<td>Human–human PPI benchmark</td>
<td>widely used in DL PPI papers</td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
<div id="tools-and-servers-summary" class="section level2 unnumbered hasAnchor">
<h2>7.4 Tools and Servers Summary<a href="references.html#tools-and-servers-summary" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This section summarizes the computational tools introduced in Chapters 2–6.</p>
<div id="docking-tools-chapter-2" class="section level3 unnumbered hasAnchor">
<h3><strong>Docking Tools (Chapter 2)</strong><a href="references.html#docking-tools-chapter-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Type</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>HADDOCK</strong></td>
<td>Flexible/data-driven docking</td>
<td><a href="https://wenmr.science.uu.nl/haddock2.4" class="uri">https://wenmr.science.uu.nl/haddock2.4</a></td>
</tr>
<tr class="even">
<td><strong>HDOCK Server</strong></td>
<td>Hybrid docking</td>
<td><a href="http://hdock.phys.hust.edu.cn" class="uri">http://hdock.phys.hust.edu.cn</a></td>
</tr>
<tr class="odd">
<td><strong>ClusPro</strong></td>
<td>Rigid-body docking</td>
<td><a href="https://cluspro.bu.edu" class="uri">https://cluspro.bu.edu</a></td>
</tr>
<tr class="even">
<td><strong>ZDOCK</strong></td>
<td>FFT-based docking</td>
<td><a href="https://zdock.umassmed.edu" class="uri">https://zdock.umassmed.edu</a></td>
</tr>
<tr class="odd">
<td><strong>GRAMM</strong></td>
<td>Grid-based docking</td>
<td><a href="http://vakser.bioinformatics.ku.edu/resources/gramm" class="uri">http://vakser.bioinformatics.ku.edu/resources/gramm</a></td>
</tr>
<tr class="even">
<td><strong>pyDock</strong></td>
<td>Scoring only</td>
<td><a href="https://life.bsc.es/pid/pydock" class="uri">https://life.bsc.es/pid/pydock</a></td>
</tr>
</tbody>
</table>
</div>
<div id="structure-analysis-chapter-3" class="section level3 unnumbered hasAnchor">
<h3><strong>Structure Analysis (Chapter 3)</strong><a href="references.html#structure-analysis-chapter-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="27%" />
<col width="44%" />
<col width="27%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Purpose</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>PDBePISA</strong></td>
<td>Interface &amp; assembly analysis</td>
<td><a href="https://www.ebi.ac.uk/pdbe/pisa" class="uri">https://www.ebi.ac.uk/pdbe/pisa</a></td>
</tr>
</tbody>
</table>
</div>
<div id="sequence-knowledgebased-models-chapter-4" class="section level3 unnumbered hasAnchor">
<h3><strong>Sequence / Knowledge–Based Models (Chapter 4)</strong><a href="references.html#sequence-knowledgebased-models-chapter-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Type</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>STRING</strong></td>
<td>Network-based PPI</td>
<td><a href="https://string-db.org" class="uri">https://string-db.org</a></td>
</tr>
<tr class="even">
<td><strong>PrePPI</strong></td>
<td>Hybrid sequence/template</td>
<td><a href="https://honiglab.c2b2.columbia.edu/PrePPI" class="uri">https://honiglab.c2b2.columbia.edu/PrePPI</a></td>
</tr>
<tr class="odd">
<td><strong>PPI-Miner</strong></td>
<td>Literature mining</td>
<td><a href="http://caps.ncbs.res.in/ppiminer" class="uri">http://caps.ncbs.res.in/ppiminer</a></td>
</tr>
<tr class="even">
<td><strong>COTH</strong></td>
<td>Template-based threading</td>
<td><a href="https://zhanggroup.org/COTH" class="uri">https://zhanggroup.org/COTH</a></td>
</tr>
</tbody>
</table>
</div>
<div id="deep-learning-models-chapter-5" class="section level3 unnumbered hasAnchor">
<h3><strong>Deep Learning Models (Chapter 5)</strong><a href="references.html#deep-learning-models-chapter-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Type</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>D-SCRIPT</strong></td>
<td>Sequence → contact map → PPI</td>
<td><a href="https://github.com/flatironinstitute/DSCRIPT" class="uri">https://github.com/flatironinstitute/DSCRIPT</a></td>
</tr>
<tr class="even">
<td><strong>DeepPurpose</strong></td>
<td>Multi-encoder deep model</td>
<td><a href="https://github.com/kexinhuang12345/DeepPurpose" class="uri">https://github.com/kexinhuang12345/DeepPurpose</a></td>
</tr>
<tr class="odd">
<td><strong>MAPE-PPI</strong></td>
<td>Multi-view attention DL model</td>
<td><a href="https://github.com/LirongWu/MAPE-PPI" class="uri">https://github.com/LirongWu/MAPE-PPI</a></td>
</tr>
</tbody>
</table>
</div>
<div id="structure-generation-tools-chapter-6" class="section level3 unnumbered hasAnchor">
<h3><strong>Structure-Generation Tools (Chapter 6)</strong><a href="references.html#structure-generation-tools-chapter-6" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<table>
<colgroup>
<col width="27%" />
<col width="44%" />
<col width="27%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Purpose</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>AlphaFold-Multimer</strong></td>
<td>Complex structure prediction</td>
<td><a href="https://github.com/deepmind/alphafold" class="uri">https://github.com/deepmind/alphafold</a></td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
<div id="references-1" class="section level2 unnumbered hasAnchor">
<h2>7.5 References<a href="references.html#references-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>HADDOCK</strong></p>
<p>Dominguez, C., Boelens, R., &amp; Bonvin, A. M. J. J. (2003). HADDOCK: A protein−protein docking approach based on biochemical or biophysical information. Journal of the American Chemical Society, 125(7), 1731–1737. <a href="https://doi.org/10.1021/ja026939x" class="uri">https://doi.org/10.1021/ja026939x</a></p>
<p>van Zundert, G. C. P., Rodrigues, J. P. G. L. M., Trellet, M., Schmitz, C., Kastritis, P. L., Karaca, E., Melquiond, A. S. J., van Dijk, M., de Vries, S. J., &amp; Bonvin, A. M. J. J. (2016). The HADDOCK2.2 Web server: User-Friendly integrative modeling of biomolecular complexes. Journal of Molecular Biology, 428(4), 720–725. <a href="https://doi.org/10.1016/j.jmb.2015.09.014" class="uri">https://doi.org/10.1016/j.jmb.2015.09.014</a></p>
<p><strong>HDOCK Server</strong></p>
<p>Yan, Y., Zhang, D., Zhou, P., Li, B., &amp; Huang, S.-Y. (2017). HDOCK: A web server for protein–protein and protein–DNA/RNA docking based on a hybrid strategy. Nucleic Acids Research, 45(W1), W365–W373. <a href="https://doi.org/10.1093/nar/gkx407" class="uri">https://doi.org/10.1093/nar/gkx407</a></p>
<p><strong>ClusPro</strong></p>
<p>Comeau, S. R., Gatchell, D. W., Vajda, S., &amp; Camacho, C. J. (2004). ClusPro: An automated docking and discrimination method for the prediction of protein complexes. Bioinformatics, 20(1), 45–50. <a href="https://doi.org/10.1093/bioinformatics/btg371" class="uri">https://doi.org/10.1093/bioinformatics/btg371</a></p>
<p>Kozakov, D., Hall, D. R., Xia, B., Porter, K. A., Padhorny, D., Yueh, C., Beglov, D., &amp; Vajda, S. (2017). The ClusPro web server for protein–protein docking. Nature Protocols, 12(2), 255–278. <a href="https://doi.org/10.1038/nprot.2016.169" class="uri">https://doi.org/10.1038/nprot.2016.169</a></p>
<p><strong>ZDOCK</strong></p>
<p>Chen, R., Li, L., &amp; Weng, Z. (2003). ZDOCK: An initial‐stage protein‐docking algorithm. Proteins: Structure, Function, and Bioinformatics, 52(1), 80–87. <a href="https://doi.org/10.1002/prot.10389" class="uri">https://doi.org/10.1002/prot.10389</a></p>
<p><strong>GRAMM (Vakser Lab)</strong></p>
<p>Singh, A., Copeland, M. M., Kundrotas, P. J., &amp; Vakser, I. A. (2023). Gramm: A webserver for free and template-based protein docking. Biophysical Journal, 122(3), 47a. <a href="https://doi.org/10.1016/j.bpj.2022.11.463" class="uri">https://doi.org/10.1016/j.bpj.2022.11.463</a></p>
<p><strong>pyDock</strong></p>
<p>Jiménez-García, B., Pons, C., &amp; Fernández-Recio, J. (2013). pyDockWEB: A web server for rigid-body protein–protein docking using electrostatics and desolvation scoring. Bioinformatics, 29(13), 1698–1699. <a href="https://doi.org/10.1093/bioinformatics/btt262" class="uri">https://doi.org/10.1093/bioinformatics/btt262</a></p>
<p><strong>PDBePISA</strong></p>
<p>Krissinel, E., &amp; Henrick, K. (2007). Inference of macromolecular assemblies from crystalline state. Journal of Molecular Biology, 372(3), 774–797. <a href="https://doi.org/10.1016/j.jmb.2007.05.022" class="uri">https://doi.org/10.1016/j.jmb.2007.05.022</a></p>
<p><strong>STRING</strong></p>
<p>Szklarczyk, D., Gable, A. L., Lyon, D., Junge, A., Wyder, S., Huerta-Cepas, J., Simonovic, M., Doncheva, N. T., Morris, J. H., Bork, P., Jensen, L. J., &amp; Mering, C. von. (2018). STRING v11: Protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Research, 47(D1), D607–D613. <a href="https://doi.org/10.1093/nar/gky1131" class="uri">https://doi.org/10.1093/nar/gky1131</a></p>
<p><strong>PrePPI</strong></p>
<p>Petrey, D., Zhao, H., Trudeau, S., Murray, D., &amp; Honig, B. (2023). PrePPI: A structure informed proteome-wide database of protein-protein interactions. Cold Spring Harbor Laboratory. <a href="https://doi.org/10.1101/2023.02.27.530276" class="uri">https://doi.org/10.1101/2023.02.27.530276</a></p>
<p><strong>PPI-Miner</strong></p>
<p>PPI-Miner: A structure and sequence motif co-driven proteinprotein interaction mining and modeling computational method. (n.d.). <a href="https://doi.org/10.1021/acs.jcim.2c01033.s001" class="uri">https://doi.org/10.1021/acs.jcim.2c01033.s001</a></p>
<p><strong>D-SCRIPT</strong></p>
<p>Sledzieski, S., Singh, R., Cowen, L., &amp; Berger, B. (2021). D-SCRIPT translates genome to phenome with sequence-based, structure-aware, genome-scale predictions of protein-protein interactions. Cell Systems, 12(10), 969-982.e6. <a href="https://doi.org/10.1016/j.cels.2021.08.010" class="uri">https://doi.org/10.1016/j.cels.2021.08.010</a></p>
<p><strong>DeepPurpose</strong></p>
<p>Huang, K., Fu, T., Glass, L. M., Zitnik, M., Xiao, C., &amp; Sun, J. (2020). DeepPurpose: A deep learning library for drug–target interaction prediction. Bioinformatics, 36(22–23), 5545–5547. <a href="https://doi.org/10.1093/bioinformatics/btaa1005" class="uri">https://doi.org/10.1093/bioinformatics/btaa1005</a></p>
<p><strong>MAPE-PPI</strong></p>
<p>Wu, L., Tian, Y., Huang, Y., Li, S., Lin, H., Chawla, N. V., &amp; Li, S. Z. (2024, February 22). MAPE-PPI: Towards effective and efficient protein-protein interaction prediction via microenvironment-aware protein embedding. arXiv.Org. <a href="https://arxiv.org/abs/2402.14391" class="uri">https://arxiv.org/abs/2402.14391</a></p>
<p><strong>AlphaFold-Multimer</strong></p>
<p>Adriaenssens, E. (2024). Protein structure prediction with AlphaFold-Multimer v1. ZappyLab, Inc. <a href="https://doi.org/10.17504/protocols.io.81wgbz25qgpk/v1" class="uri">https://doi.org/10.17504/protocols.io.81wgbz25qgpk/v1</a></p>
<p><strong>DockQ</strong></p>
<p>Basu, S., &amp; Wallner, B. (2016). DockQ: A quality measure for protein-protein docking models. PLOS ONE, 11(8), e0161879. <a href="https://doi.org/10.1371/journal.pone.0161879" class="uri">https://doi.org/10.1371/journal.pone.0161879</a></p>
<hr />

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="structure-generation.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/HayesYoung58/Protein-Protein-Interaction-PPI-Prediction-Methods-Summary/edit/main/07-references.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": {},
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
